tapir/accounts/templates/accounts/email/create_account_reminder.body.default.html

```diff
@@ -0,0 +1,47 @@
+{% extends "core/email_base.html" %}
+
+{% load i18n %}
+{% load utils %}
+
+{% block body %}
+    {% get_display_name_short member_infos as display_name_short %}
+    {% blocktranslate with display_name_short=display_name_short %}
+        <p>Hello {{ display_name_short }},</p>
+
+        <p>We noticed that you have been a member of SuperCoop for a month now and have not created a Tapir account to sign up for your first shift.</p>
```

Comment:
Using the text from here: https://docs.google.com/document/d/1l3vOpF1HnvQFJtJaZ22oGgcPaIb3SULm5rsaVBNSODE/edit